src/needless_bool.rs

```diff
@@ -37,7 +37,7 @@ impl LintPass for NeedlessBool {
                               "you can reduce your if statement to its predicate"); },
                 (Option::Some(false), Option::Some(true)) => {
                     span_lint(cx, NEEDLESS_BOOL, e.span,
```

Comment:
Perhaps here we should improve the error message further with snippets? "predicate" may not be understood by all.

Something like "you can reduce this if statement to {snippet}"